frontend/src/metabase/static-viz/components/NoResults/NoResults.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactElement } from "react";
+import { Group } from "@visx/group";
+import { Text } from "@visx/text";
+import { ICON_SIZE, ICON_MARGIN, LAYOUT } from "./constants";
+
+const NoResults = (): ReactElement => {
+  const innerWidth = LAYOUT.width - LAYOUT.margin * 2;
+  const innerHeight = LAYOUT.height - LAYOUT.margin * 2;
+  const centerX = LAYOUT.margin + innerWidth / 2;
+  const centerY = LAYOUT.margin + innerHeight / 2;
+  const iconPositionY = centerY - (ICON_SIZE * 2) / 3;
+
+  return (
+    <svg width={LAYOUT.width} height={LAYOUT.height}>
+      {/* based on no_results.svg */}
```

Comment:
Can't you simply import the SVG?

Reply:
`[NoResults.tsx]` generate a `svg` (as for the `StaticChart` component) to be rendered as a `png` and it is composed of the no results `icon` and the `text`.
`no_results.svg` only contains the icon without any text.

Reply:
Is there a chance to inject the text dynamically here?
Beside, the text likely needs to support i18n (look for ttag to find some examples).

Reply:
@maz perhaps do you know if we have a precedence of composing an SVG icon with a caption/text?

Reply:
@ariya 

Since you can't manipulate svg `children` when loaded in React using `svgr`, I used the svg `image` tag: https://github.com/metabase/metabase/pull/20891/files#diff-12f17cba6dc014361b3cdfe519e2d00f424f0d1b022a1b53f4a3896e116b1203R17.

Reply:
Nice trick @losrebellos :+1: Does it still appear properly in Slack/email? (IOW: the backend still renders the SVG as PNG faithfully)

Reply:
@howonlee @ariya 

It has been tested locally FE side but atm, when there is no data, the rendering of the `png` is skipped by the BE.
This needs to be removed before we can push forwards with more tests.

Reply:
@ariya It does work but the svg image is loaded as attachment. I believe it's not great experience for the user and we should use inline svg for this.

<img width="1695" alt="Screenshot 2022-03-16 at 17 48 11" src="https://user-images.githubusercontent.com/8542534/158617600-6c57d220-f4e6-45be-819b-0fdbf9e3d56d.png">
<img width="965" alt="Screenshot 2022-03-16 at 17 48 23" src="https://user-images.githubusercontent.com/8542534/158617630-ea070415-02ac-430c-b371-5849819b7598.png">

Reply:
@ranquild At least with subscription sent to Slack, all the charts appear as images anyway. This is completely different with subscription sent to email?

Reply:
The case with `no results` seems to be rendered differently